src/focus.service.ts

```diff
@@ -509,6 +509,22 @@ export class FocusService {
   private isFocusable(el: HTMLElement) {
     const role = el.getAttribute('role');
     const tabIndex = el.tabIndex;
+    let visible = el.offsetParent !== null;
+
+    for (let parent = el; parent !== this.root; parent = parent.parentElement) {
+      if (parent.classList.contains('hidden')) {
```

Comment:
Unfortunately that's not working either. I also tried opacity === 0 but that only works on the parent nav, not the tabs